tee-worker/ts-tests/integration-tests/identity.test.ts

```diff
@@ -24,6 +24,33 @@ import type { LitentryValidationData, Web3Network } from 'parachain-api';
 import type { TransactionSubmit } from './common/type-definitions';
 import type { HexString } from '@polkadot/util/types';
 import { ethers } from 'ethers';
+import { sendRequest } from './common/call';
+import * as base58 from 'micro-base58';
+
+async function getWorkerAddress(base58mrEnclave: string, context: any) : Promise<string> {
+    const requestAcc = { jsonrpc: '2.0', method: 'author_getEnclaveAccountId', params: [base58mrEnclave], id: 1 };
+    const resAcc = await sendRequest(context.tee, requestAcc, context.api);
+    const workerAcc = resAcc.value.toHex();
+    console.log("worker address value", workerAcc);
+    return workerAcc;
+}
+
+async function getNonce(base58mrEnclave: string, workerAddr: string, context: any) : Promise<number> {
+    const request = { jsonrpc: '2.0', method: 'author_getNextNonce', params: [base58mrEnclave, workerAddr], id: 1 };
+    const res = await sendRequest(context.tee, request, context.api);
+    const u8aValue = res.value.toU8a();
+    const len = u8aValue.length;
+    let nonce = 0;
+    if(len > 0) {
+        for( let i = len - 1; i > 0; i--){
+            nonce *= 16;
+            nonce += u8aValue[i];
+        }
+    }
+    console.log("res value",u8aValue);
+    console.log("nonce is:", nonce);
+    return nonce;
+}
```

Comment:
I'm confused -- does this give a different value than `res.value.toHuman()`?

Reply:
u8a is array like this [ 16, 15, 0, 0, 0 ].  toHuman() is like this: 0x0f000000. Which would you perfer to print?

Reply:
What I mean is, why do we need to decode the nonce by hand instead of using the baked-in functionality?